integration/swarm/swarm.bats

```diff
@@ -66,6 +70,29 @@ setup() {
 	done
 }
 
+@test "obtain hostname of the replicas" {
+	# this will help to obtain the hostname of
+	# the replicas from the curl
+	unset http_proxy
+	check_status=$($DOCKER_EXE ps --filter status=running --format "{{.ID}}" | wc -l)
+	if [ "$check_status" -lt "$number_of_replicas" ]; then
+		sleep 1
+	else
+		break
+	fi
+	for i in `seq 0 $((number_of_replicas-1))`; do
+		# this will help that bat does not exit incorrectly
+		# when curl fails in one of the attemps
+		set +e
+		REPLICAS[$i]="$(curl $url)"
```

Comment:
I think this should probably use `--connect-timeout=$timeout_secs` (where `$timeout_secs=60` or something). Also, what about using one of the `--retry*` options to avoid having to use the `+e`?